src/app/components/modules/progress/progress.component.ts

```diff
@@ -222,6 +222,10 @@ export class ProgressComponent extends SubscriptionsComponent implements OnInit,
 		// Define nowTime just to make things clearer
 		const nowTime = this.today.getTime();
 
+		//End of School constant created for DRY
+
+		const SchoolDayEnd315 = moment(this.today).startOf('day').hours(15).minutes(15);
```

Comment:
We also use camel case for variables but without capitalizing the first letter of the variable